### PR TITLE
Alertmanager configuration process updates

### LINKED ISSUE #66: Invalid procedure in "Creating an alert route in Alertmanager"
In `doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-route-in-alertmanager.adoc` at step 3, the deletion of `alertmanager-stf-default` is invalid because that `Secret` object is managed by the Service Telemetry Operator.

The proper procedure is to `oc edit servicetelemetry stf-default` and override the default `alertmanager-stf-default` secret with the `alertmanagerConfigManifest` parameter.

See https://github.com/infrawatch/service-telemetry-operator/blob/master/roles/servicetelemetry/tasks/component_alertmanager.yml#L1-L36 for how this is managed in the Operator.

### PR DESCRIPTION
**Pull request by leifmadsen**
Closes #66

### CODE REVIEW

File: doc-Service-Telemetry-Framework/modules/proc_configuring-custom-alerts.adoc [hunk 1]
```diff
@@ -23,12 +23,12 @@
 [id="configuring-custom-alerts_{context}"]
 = Configuring custom alerts
 
-You can edit the PrometheusRule that you created in <<creating-an-alert-rule-in-prometheus_advanced-features>>:
+Custom alerts can be added to the `PrometheusRule` object created in <<creating-an-alert-rule-in-prometheus_advanced-features>>:
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_configuring-custom-alerts.adoc:26]:
```suggestion
You can add custom alerts to the `PrometheusRule` object that you created in <<creating-an-alert-rule-in-prometheus_advanced-features>>:
```

File: doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-route-in-alertmanager.adoc [hunk 2]
```diff
@@ -23,109 +23,118 @@
...
 
-. Restart Alertmanager:
+. To verify the configuration has been loaded into Alertmanager, create a pod with access to `curl`:
 +
 ----
-oc delete pod alertmanager-stf-default-0
+oc run curl --generator=run-pod/v1 --image=radial/busyboxplus:curl -i --tty
 ----
 
+. Then run `curl` against the `alertmanager-operated` service to retrieve the status and `configYAML` contents and review the supplied configuration matches the configuration loaded into Alertmanager:
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-route-in-alertmanager.adoc:117]:
```suggestion
. Run `curl` against the `alertmanager-operated` service to retrieve the status and `configYAML` contents and review the supplied configuration matches the configuration loaded into Alertmanager:
```

File: doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-route-in-alertmanager.adoc [hunk 3]
```diff
@@ -23,109 +23,118 @@
...
+----
 
-. To validate that the manifest loaded correctly, run the following command:
+. Validate the `configYAML` field contains the expected changes, then exit from the pod:
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-route-in-alertmanager.adoc:125]:
```suggestion
. Validate the `configYAML` field contains the expected changes and exit from the pod:
```

File: doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-route-in-alertmanager.adoc [hunk 4]
```diff
@@ -23,109 +23,118 @@
...
 +
 ----
-oc get prometheusrules
+[ root@curl:/ ]$ exit
 ----
 
-. To validate that the rule loaded into Prometheus, run the following command:
+. Clean up the environment by delete the `curl` pod:
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-route-in-alertmanager.adoc:131]:
```suggestion
. To clean up the environment, delete the `curl` pod:
```

File: doc-Service-Telemetry-Framework/modules/proc_configuring-custom-alerts.adoc (hunk 1 above)
Comment (leifmadsen) [reply doc-Service-Telemetry-Framework/modules/proc_configuring-custom-alerts.adoc:26]: Huh I was specifically avoiding that narrator position. Maybe I've misunderstood what voice I'm aiming for this whole time?

Comment (JoanneOFlynn2018) [reply doc-Service-Telemetry-Framework/modules/proc_configuring-custom-alerts.adoc:26]: It's very easy to fall into passive voice but active voice is always the preferred voice. It reduces confusion and is more prescriptive

**Pull request merged**